diesel/src/r2d2.rs

```diff
@@ -3,6 +3,8 @@
 extern crate r2d2;
 
 pub use self::r2d2::*;
+/// Rexport r2d2 crate's `Error` to prevent shadowing by this module's `Error` type.
```

Comment:
This reads more like a code comment than documentation. Can we say something like "A re-export of `r2d2::Error`, which is only used by methods on `r2d2::Pool`"?